misk-jdbc/src/main/kotlin/misk/jdbc/DataSourceConfig.kt

```diff
@@ -59,6 +59,7 @@ data class DataSourceConfig @JvmOverloads constructor(
   val connection_idle_timeout: Duration? = null,
   val connection_max_lifetime: Duration = Duration.ofMinutes(1),
   val query_timeout: Duration? = Duration.ofMinutes(1),
+  val keepalive_time: Duration = Duration.ofSeconds(0),
```

Comment:
Is zero infinite? Is that the default?

Reply:
0 is the default, which translates to disabled (sourced from the README in https://github.com/brettwooldridge/HikariCP)